source/features/clean-dashboard.css

```diff
@@ -16,7 +16,7 @@
 
 /* Add background and border Whole list */
 .rgh-clean-dashboard .news > .f4 + div {
-	background-color: var(--color-bg-canvas, #fff);
+	background-color: var(--color-canvas-default, var(--color-bg-canvas));
```

Comment:
Also drops the default hardcoded white. Now it will default to transparent instead, which is almost good anyway.